src/components/search/search-result.js

```diff
@@ -3,12 +3,29 @@ import { Link } from 'gatsby'
 import DetailText from '~components/common/detail-text'
 import resultStyle from './search-result.module.scss'
 
-const SearchResult = ({ title, url, children = false }) => (
+const SearchResult = ({
+  title,
+  url,
+  publishDate,
+  author,
```

Comment:
`publishDate` & `author` could use an empty string as default, so you're type safe and can use `publishDate.length` to make it more obvious that these will always be strings (either the actual value or the default value)